eng/testing/WasmRunnerTemplate.cmd

```diff
@@ -0,0 +1,45 @@
+@echo off
+setlocal enabledelayedexpansion
+
+set EXECUTION_DIR=%~dp0
+set SCENARIO=%3
+
+if [%HELIX_WORKITEM_UPLOAD_ROOT%] == [] (
+	set XHARNESS_OUT="%EXECUTION_DIR%xharness-output"
```

Comment:
nit: I think most of the other Windows .cmd files uses spaces for indentation instead of tabs